src/components/dropdown/dropdown.css

```
@@ -0,0 +1,36 @@
+.base-dropdown {
+    position: absolute;
+    padding: 1rem;
+    width: 250px;
+    border-radius: 1rem;
+}
+
+.base-dropdown::before {
+    content: '';
+    position: absolute;
+    transform: rotate(45deg);
+}
+
+.light.solid {
+    background: var(--base-white);
+    border-color: var(--blue-base);
+    color: var(--base-black);
+  }
+  
+  .light.outline {
+    background: transparent;
+    border-color: var(--blue-base);
+    color: var(--blue-base);
+  }
+  
+  .dark.solid {
+    background: var(--base-black);
+    border-color: var(--blue-inverse);
+    color: var(--base-white);
+  }
+  
+  .dark.outline {
+    background: transparent;
+    border-color: var(--blue-inverse);
+    color: var(--blue-inverse);
+}
```

```suggestion
.base-dropdown.light {
  background: var(--base-white);
  color: var(--base-black);
}

.base-dropdown.dark {
  background: var(--base-black);
  color: var(--base-white);
}
```

Eu falo um pouco do problema com o `outline` e `solid`  no comentário mais abaixo, mas a ideia é que o componente do dropdown não tem essas 2 versões diferentes pra solid e outline. Isso era coisa do botão. No figma tem só versões do componente em light e dark mode. Além disso, como a gente tá usando as classes `.light` e `.dark` pra todos os componentes, lembra de usar a classe geral do componente antes, esse `.base-dorpdown`, senão vai dar conflito entre os estilos em diferentes componentes .